openassessment/xblock/submission_mixin.py

```diff
@@ -44,6 +44,7 @@ class SubmissionMixin(object):
         'msh1xml', 'msh2xml', 'action', 'apk', 'app', 'bin', 'command', 'csh',
         'ins', 'inx', 'ipa', 'isu', 'job', 'mst', 'osx', 'out', 'paf', 'prg',
         'rgs', 'run', 'sct', 'shb', 'shs', 'u3p', 'vbscript', 'vbe', 'workflow',
+        'htm', 'html',
```

Comment:
Yes. I had accidentally rebased this branch against master without rebasing ora-staff-grading first. The diff has gone away now.